osu.Game/Online/Metadata/MultiplayerPlaylistItemStats.cs

```diff
@@ -25,5 +25,14 @@ public class MultiplayerPlaylistItemStats
         /// </summary>
         [Key(1)]
         public long[] TotalScoreDistribution { get; set; } = new long[TOTAL_SCORE_DISTRIBUTION_BINS];
+
+        /// <summary>
+        /// The cumulative total of all passing scores (across all users) in the playlist so far.
+        /// </summary>
+        [Key(2)]
+        public long TotalPlaylistScore { get; set; }
```

Comment:
Slight naming nit but the class is named `MultiplayerPlaylistItemStats` because it was kinda designed to potentially be usable for standard playlists. So being hypercorrect this could probably be better named as <code>TotalPlaylist<strong>Item</strong>Score</code>, but not sure it's a huge deal.

Reply:
Fair, are you okay with `CumulativeScore` instead to better clarify and remove redundant item part (see class name)

Reply:
Yep, works for me.